cmd/zetae2e/local/local.go

```diff
@@ -318,6 +323,51 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 
 	logger.Print("✅ e2e tests completed in %s", time.Since(testStartTime).String())
 
+	if testMigration {
+		migrationCtx, cancel := context.WithCancel(context.Background())
+		deployerRunner.CtxCancel = cancel
+		migrationStartTime := time.Now()
+
+		logger.Print("🏁 starting tss migration")
+		response, err := deployerRunner.CctxClient.LastZetaHeight(migrationCtx, &crosschaintypes.QueryLastZetaHeightRequest{})
+		if err != nil {
+			logger.Error("cctxClient.LastZetaHeight error: %s", err)
+			panic(err)
+		}
+		err = zetaTxServer.UpdateKeygen(response.Height)
+		if err != nil {
+			panic(err)
+		}
+		waitKeygenHeight(migrationCtx, deployerRunner.CctxClient, deployerRunner.ObserverClient, logger, 0)
+		// migration test is a blocking thread, we cannot run other tests in parallel
+		eg.Go(migrationTestRoutine(conf, deployerRunner, verbose, e2etests.TestMigrateTssEthName))
+
+		if err := eg.Wait(); err != nil {
+			deployerRunner.CtxCancel()
+			logger.Print("❌ %v", err)
+			logger.Print("❌ tss migration failed")
+			os.Exit(1)
+		}
+
+		logger.Print("✅ migration tests completed in %s sleeping for 30 secs ", time.Since(migrationStartTime).String())
+		logger.Print("🏁 starting post migration tests")
+
+		tests := []string{
+			e2etests.TestBitcoinWithdrawSegWitName,
+			e2etests.TestEtherWithdrawName,
+		}
```

Comment:
Testing just Ethereum and bitcoin Withdraw for now .
Will be adding an entire rerun , in the future when TSS update to Connector and ERC20 has been added into the test